sass/blocks/intro/_intro.scss

```diff
@@ -1,23 +1,34 @@
 .intro {
   height: 39rem;
   overflow: hidden;
-  background: url("../images/intro-welcome.png"),
-    url("../images/intro-sedonaa.png"),
-    url("../images/intro-beautiful.png"),
-    url("../images/intro-background.png");
+  background: url("../img/text-sedona-index.svg"),
+    url("../img/bg-intro-triangle-mobile.svg"),
+    url("../img/bg-back-mobile@1x.jpg");
   background-repeat: no-repeat,
-    no-repeat,
     no-repeat,
     no-repeat;
-  background-size: 156px 52px,
-    282px 49px,
-    180px 46px,
+  background-size: 293px 212px,
+    100% 100%,
     cover;
-  background-position: 49% 27%,
-    48% 51%,
-    46% 74%,
+  background-position: 49% 51%,
+    0 182px,
     center;
 
+  @media (min-resolution: 144dpi), (min-resolution: 1.5dppx) {
```

Comment:
This looks overcomplicated and lots of duplications. You don't need to repeat background-repeat and background-size properties. just replace images.